Docker compose Invalid subnet

## Bug Report

### Setup

**CometBFT version** (use `cometbft version` or `git rev-parse --verify HEAD` if installed from source):

2c6e0c160da

**Have you tried the latest version**: yes/no

**ABCI app** (name for built-in, URL for self-written if it's publicly available):

**Environment**:
- **OS** (e.g. from /etc/os-release): Ubuntu
- **Install tools**:
- **Others**:

```
docker-compose version 1.29.2, build unknown
docker-py version: 5.0.3
CPython version: 3.10.12
OpenSSL version: OpenSSL 3.0.2 15 Mar 2022
```

**node command runtime flags**:

N/A

### What happened?

```text
Creating network "cometbft_localnet" with driver "bridge"
ERROR: invalid network config:
invalid subnet 192.167.10.0/16: it should be 192.167.0.0/16
make: *** [Makefile:304: localnet-start] Error 1
```

### What did you expect to happen?

The command spin up a 4-node network.

### How to reproduce it

Follow the instructions on here: https://docs.cometbft.com/v0.38/networks/docker-compose

### Additional information

Following the error's advice seemed to work:

```diff
diff --git a/docker-compose.yml b/docker-compose.yml
index fc339ede8..825dccac1 100644
--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -67,4 +67,4 @@ networks:
     ipam:
       driver: default
       config:
-        - subnet: 192.167.10.0/16
+        - subnet: 192.167.0.0/16
```

--- a/docker-compose.yml
+++ b/docker-compose.yml
@@ -67,4 +67,4 @@ networks:
     ipam:
       driver: default
       config:
-        - subnet: 192.167.10.0/16
+        - subnet: 192.167.0.0/16

